private/bufpkg/bufimage/build_image.go

```diff
@@ -449,3 +458,15 @@ func (r *resolverForBuildResult) FindEnumByName(enum protoreflect.FullName) (pro
 	}
 	return dynamicpb.NewEnumType(enumDescriptor), nil
 }
+
+func addFileToMap(allFiles map[string]linker.File, file linker.File) {
```

Comment:
keyToValue for map names - will always comment this for bufbuild/buf PRs

Reply:
done